pkgs/development/python-modules/speechbrain/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, huggingface-hub
+, hyperpyyaml
+, joblib
+, lib
+, numpy
+, packaging
+, pythonOlder
+, sentencepiece
+, scipy
+, torchaudio
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "speechbrain";
+  version = "0.5.14";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "speechbrain";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-r1q7JO+H7ynfrzlihRTY0PtMGmvwm98BHUZV534ABXw=";
+  };
+
+  propagatedBuildInputs = [
+    huggingface-hub
+    hyperpyyaml
+    joblib
+    numpy
+    packaging
+    sentencepiece
+    scipy
+    torchaudio
```

Comment:
Is `torch` dependency left out on purpose? (I see it is specified in `setup.py`.)

Reply:
It didn't complain about it being missing. Actually I think that it is thanks to `torchaudio` carrying it.
I added it explicitly though.

Reply:
Please don't rely on other dependencies bringing in required dependencies. This will cause failures when other dependencies change their dependencies.